### ISSUE TITLE
Requirements Question - Connectivity

### ISSUE THREAD

**Issue by pmcdougal** [Rytiggy/api-watchface#18]
Huge thank you for developing this.

To confirm, are the only requirements an Ionic/Versa and a G5 + smartphone with Bluetooth enabled on both? Does WiFi need to be enabled on either device? If so will Androids default shut down of WiFi when the phone is sleeping interrupt the interconnectivity?

Thanks!

**Comment by Rytiggy** [comment]
Yep, that's the only requirement, but you can use any CGM. I use the Guardian Link with my 670g. I haven't seen any interruptions yet but this is a good thought. I'll have to test it.
